tools/prereqs/setup_terraform.sh

```diff
@@ -30,8 +30,10 @@ cd $tmp_dir
 ostype=`os_type`
 if [ "$ostype" == "linux" ]; then
     arch="linux_amd64"
+    sudo apt-get install unzip
 elif [ "$ostype" == "macos" ]; then
     arch="darwin_amd64"
+    brew install unzip
```

Comment:
Makes an assumption brew is installed.  Also, unzip comes preinstalled on every Mac I have ever had.